File: Pak Dengklek Seorang Deadliner/PakDengklekSeorangDeadliner.java
import java.util.Scanner;

public class PakDengklekSeorangDeadliner
{
    public static void main (String [] args)
    {
        Scanner scan = new Scanner (System.in);
        int N = scan.nextShort ();
        
        int jam = N / 3600;
        N %= 3600;
        int menit = N / 60;
        N %= 60;

        System.out.println (jam);
        System.out.println (menit);
        System.out.print (N);

        scan.close ();
    }    
}

File: Pak Dengklek Seorang Deadliner/Pak-Dengklek-Seorang-Deadliner.cpp
#include <iostream>

int main ()
{
    short N = 0, menit = 0, jam = 0;
    std::cin >> N;

    jam = N / 3600;
    N %= 3600;
    menit = N / 60;
    N %= 60;

    std::cout <<  jam << "\n" << menit << "\n" << N;

    return 0;
}